src/components/Datagrid/ColumnPicker/ColumnPicker.tsx

```diff
@@ -0,0 +1,107 @@
+import {
+  Box,
+  Button,
+  Popover,
+  sprinkles,
+  TableEditIcon,
+  Text,
+  vars,
+} from "@saleor/macaw-ui/next";
+import React, { useState } from "react";
+import { FormattedMessage } from "react-intl";
+
+import { AvailableColumn } from "../types";
+import { ColumnPickerCategories } from "./ColumnPickerCategories";
+import { ColumnPickerDynamicColumns } from "./ColumnPickerDynamicColumns";
+import { ColumnPickerStaticColumns } from "./ColumnPickerStaticColumns";
+import messages from "./messages";
+import { ColumnCategory } from "./useColumns";
+
+export interface ColumnPickerProps {
+  staticColumns: AvailableColumn[];
+  dynamicColumns?: AvailableColumn[];
+  selectedColumns: string[];
+  columnCategories?: ColumnCategory[];
+  columnPickerSettings?: string[];
+  onSave: (columns: string[]) => void;
+  onDynamicColumnSelect?: (columns: string[]) => void;
+}
+
+export const ColumnPicker: React.FC<ColumnPickerProps> = ({
+  staticColumns,
+  selectedColumns,
+  columnCategories,
+  dynamicColumns,
+  columnPickerSettings,
+  onDynamicColumnSelect,
+  onSave,
+}) => {
```

Comment:
Keep the same convention everywhere
```suggestion
export const ColumnPicker = ({
  staticColumns,
  selectedColumns,
  columnCategories,
  dynamicColumns,
  columnPickerSettings,
  onDynamicColumnSelect,
  onSave,
}: ColumnPickerProps) => {
```